### ISSUE TITLE
Mobile View- Showcase Page- Footer not viewable

### ISSUE THREAD

**Issue by TylerTrott** [atsign-foundation/atsign.dev#429]
*What the issue is*
When navigating to our Showcase page on mobile view, the footer does not exist at the bottom of the page.

*How to resolve this issue*
There are a few ways we could tackle this one:
- This issue may be resolved synchronously with the resolution of Issue #421 
- Fix with shortcode 
- Format proper CSS

**Comment by Sandhya312** [comment]
Hey @TylerTrott, I am a contributor to openforce'22. I want to work on this issue. Please assign this issue to me.

**Comment by hi-anusha** [comment]
Hi, I am Anusha, a contributor to OpenForce 2022. I would like to work on this issue. Also please assign this issue to me. I cloned your repository and since there are many folders I would like to know which file or folder should I open to work on this issue? Thank you.

**Comment by vermaayush680** [comment]
Hey @TylerTrott 
I would like to work on this, if it is available..

**Comment by SumitKanth** [comment]
hii @TylerTrott I want to work on this, please assign me

**Comment by nik132-eng** [comment]
Hi, I am Nikunj Rohit, a contributor to OpenForce 2022. If this issue is still open I would like to work on this issue and try better to enhance it more. I would be making a PR as soon as I am done with resolving the issue. Thank you
@yahu1031

**Comment by nik132-eng** [comment]
@yahu1031 if the issue is already solved so please don't assign it to other persons.
![image](https://user-images.githubusercontent.com/77881638/159693978-9581ec5a-a060-42e7-bacb-3608a71ea782.png)

**Comment by yahu1031** [comment]
Lemme check and update the ticket

**Comment by yahu1031** [comment]
yeah, fixed 😄  thanks for letting us know. Please mention the PR for the fix
